serverRoot/src/main/server/org/compiere/server/AdempiereServer.java

```diff
@@ -198,7 +201,19 @@ public void runNow()
 		//
 		log.fine(getName() + ": " + getStatistics());
 	}	//	runNow
-
+
+	/**
+	 * Validate if is fixed time
+	 * @return
+	 */
+	private boolean isValidForRun() {

```

Comment:
For testing - if the junit upgrades gets approved - consider making this a protected method.  It is difficult to test private methods.

Reply:
You are right, I will go